src/NuGet.Core/NuGet.Protocol/Model/PackageSearchMetadata.cs

```diff
@@ -204,5 +204,7 @@ private static IList<string> GetNonStandardLicenseIdentifiers(NuGetLicenseExpres
         /// </summary>
         [JsonProperty(PropertyName = JsonProperties.Deprecation)]
         public PackageDeprecationMetadata DeprecationMetadata { get; private set; }
+
+        public Task<PackageDeprecationMetadata> GetDeprecationMetadataAsync() => Task.FromResult(DeprecationMetadata);
```

Comment:
You can write a test for this.

Reply:
There were no existing tests for this class, so I made a simple one to check just this property.